File: src/SOP8/Base/Account/Base.Account.IBLL/Request/RequestOption.cs
namespace Base.Account.IBLL.Request
{
    public class RequestOption
    {
        private int m_nUserNo = -1;
        private string m_strCategory = null;
        private string m_strSubCategory = null;

        public int UserNo
        {
            get { return m_nUserNo; }
            set { m_nUserNo = value; }
        }

        public string Category
        {
            get { return m_strCategory; }
            set { m_strCategory = value; }
        }

        public string SubCategory
        {
            get { return m_strSubCategory; }
            set { m_strSubCategory = value; }
        }
    }
}

File: src/SOP8/Egovframework/Base/base-account-ibll/src/main/java/egovframework/base/account/ibll/request/RequestOption.java
package egovframework.base.account.ibll.request;

public class RequestOption {

    private int userNo = -1;
    private String category = null;
    private String subCategory = null;

    public int getUserNo() {
        return userNo;
    }

    public void setUserNo(int userNo) {
        this.userNo = userNo;
    }

    public String getCategory() {
        return category;
    }

    public void setCategory(String category) {
        this.category = category;
    }

    public String getSubCategory() {
        return subCategory;
    }

    public void setSubCategory(String subCategory) {
        this.subCategory = subCategory;
    }
}